checkov/common/output/report.py

```diff
@@ -272,6 +272,7 @@ def print_console(
             use_bc_ids: bool = False,
             summary_position: str = 'top',
             openai_api_key: str | None = None,
+            openai_api_type: str = "default",
```

Comment:
remove all of the `openai_api_type` parameter changes and change it to an env var with a default of `default`